templates/shared/_pro-contact-us-form.html

```diff
@@ -22,88 +22,116 @@ <h2 class="p-heading--3">If you use Ubuntu, which version(s) are you using?</h2>
                 <span class="p-checkbox__label" id="22-04">22.04 LTS</span>
               </label>
               <label class="p-checkbox">
-                <input type="checkbox" aria-labelledby="20-04" class="p-checkbox__input" value="20.04 LTS">
+                <input type="checkbox" aria-labelledby="20-04" class="p-checkbox__input" value="20.04 LTS" />
                 <span class="p-checkbox__label" id="20-04">20.04 LTS</span>
               </label>
             </div>
             <div class="col-4 u-sv3">
               <strong>LTS out of standard support</strong>
               <label class="p-checkbox">
-                <input type="checkbox" aria-labelledby="18-04" class="p-checkbox__input" value="18.04 LTS">
+                <input type="checkbox" aria-labelledby="18-04" class="p-checkbox__input" value="18.04 LTS" />
                 <span class="p-checkbox__label" id="18-04">18.04 LTS</span>
               </label>
               <label class="p-checkbox">
-                <input type="checkbox" aria-labelledby="16-04" class="p-checkbox__input" value="16.04 LTS">
+                <input type="checkbox" aria-labelledby="16-04" class="p-checkbox__input" value="16.04 LTS" />
                 <span class="p-checkbox__label" id="16-04">16.04 LTS</span>
               </label>
               <label class="p-checkbox">
-                <input type="checkbox" aria-labelledby="14-04" class="p-checkbox__input" value="14.04 LTS">
+                <input type="checkbox" aria-labelledby="14-04" class="p-checkbox__input" value="14.04 LTS" />
                 <span class="p-checkbox__label" id="14-04">14.04 LTS</span>
               </label>
             </div>
             <div class="col-4 u-sv3">
               <strong>Outdated or non-LTS releases</strong>
               <label class="p-checkbox">
-                <input type="checkbox" aria-labelledby="22-10" class="p-checkbox__input"
+                <input type="checkbox" aria-labelledby="22-10" class="p-checkbox__input /"
                   value="22.10 (interim release)">
                 <span class="p-checkbox__label" id="22-10">22.10 (interim release)</span>
               </label>
               <label class="p-checkbox">
-                <input type="checkbox" aria-labelledby="21-04" class="p-checkbox__input"
+                <input type="checkbox" aria-labelledby="21-04" class="p-checkbox__input /"
                   value="21.04 (interim release)">
                 <span class="p-checkbox__label" id="21-04">21.04 (interim release)</span>
               </label>
               <label class="p-checkbox">
-                <input type="checkbox" aria-labelledby="12-04" class="p-checkbox__input" value="12.04 LTS">
+                <input type="checkbox" aria-labelledby="12-04" class="p-checkbox__input" value="12.04 LTS" />
                 <span class="p-checkbox__label" id="12-04">12.04 LTS</span>
               </label>
             </div>
             <div class="col-4 u-sv3">
```

Comment:
Copy for this one is a bit different on the doc 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/f194cc08-3613-4de1-b64e-aafaca21d146)

Reply:
well, this is up to @Sophie-32 I assume production is the correct one?

Reply:
Sure, the obvious fix is updating the doc. @Sophie-32 could you please take a look and make any needed changes?

Reply:
Yes I'll change the copydoc to read 'confirmation page'. Its just the template says 'banner' since that's now the default, but I agree and will try to make sure its clear moving forward.

Reply:
I'm so sorry, it does need to match the copydoc. Apologies, once again I learn the importance of making sure that updates are in suggesting mode